src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -447,6 +448,19 @@ private async Task<CreationResultStatus> ExecuteAsync()
                 return CreationResultStatus.Success;
             }
 
+            try
+            {
+                bool isHiveUpdated = SyncOptionalWorkloads();
+                if (isHiveUpdated)
+                {
+                    Reporter.Output.WriteLine(LocalizableStrings.OptionalWorkloadsSynchronized);
```

Comment:
Does isHiveUpdated == true whenever any of the optional workloads packages present? This string about sync must be very annoying in this case. Could we check that installation/update has been really performed?

Reply:
Updated. Set `isHiveUpdated = true` only in case there is something new to install/update/remove